dotCMS/src/main/resources/portal.properties

```diff
@@ -305,7 +305,14 @@
     # Note that \ is replaced with \\ to work in Java.
     #
 
-    # This pattern ensures that passwords must have at least 6 characters and no white spaces.
+    # This pattern below ensures that passwords must have at least 8 characters and no white spaces.
+    #	Only the specified characters are allowed:
+    #   	1. Uppercase letters: A-Z
+    #		2. Lowercase letters: a-z
+    #		3. Digits: 0-9
+    #           4. Special characters: ! # % + : = ? @
+    # 		5. No white spaces or other characters not included in the specified set.
+    # The entire password must consist exclusively of the allowed characters from the start (^) to the end ($) of the string.
     passwords.regexptoolkit.pattern=/^[!#%+1234567890:=?@ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz]{8,}$/

```

Comment:
Better would be min set to 12 chars but no more than 64

/^[!#%+1234567890:=?@ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz]{12,64}$/